cmd/ci-operator-config-mirror/main.go

```diff
@@ -106,21 +152,27 @@ func main() {
 
 		repoInfo.Org = o.toOrg
 		rbc.Metadata.Org = o.toOrg
-		dataWithInfo := config.DataWithInfo{
+		dataWithInfoByRepo[repoInfo.Repo] = append(dataWithInfoByRepo[repoInfo.Repo], config.DataWithInfo{
 			Configuration: *rbc,
 			Info:          *repoInfo,
-		}
-
-		if err := dataWithInfo.CommitTo(o.configDir); err != nil {
-			logger.WithError(err).Fatal("couldn't create the configuration file")
-		}
+		})
 
 		return nil
 	}
 
 	if err := config.OperateOnCIOperatorConfigDir(o.configDir, callback); err != nil {
 		logrus.WithError(err).Fatal("error while generating the ci-operator configuration files")
```

Comment:
Looks like we no longer actually generate anything in `callback`, so this message needs to be changed.